### PR TITLE
CSS Answers

### PR DESCRIPTION
**Pull request by logeshpaul**
- Added a few more answers
- Formatted based on headings

### CODE REVIEW

File: CSS/README.md [hunk 1]
```diff
@@ -102,55 +102,115 @@
...
-- [ ] How is responsive design different from adaptive design?
-- [ ] Have you ever worked with retina graphics? If so, when and what techniques did you use?
+- [x] Considerations for printing webpages?
+  
+  > - Specify a Print Style Sheet `<link rel="stylesheet" href="print.css" media="print" />`
+  > - Hide Extraneous Elements `.navigation, .footer { display: none;}` 
+  > - Bump Up Font Sizes and Line-Heights `body { font-size: 120%; line-height: 130%; }`
+  > - Move to Serif Fonts `body { font-family: Georgia, "Times New Roman", Times, serif; }`
+  > - Display the destination of links & make them standout with different color `a:link:after { content: " (" attr(href) ") "; }`
+  > - Enlarge content area `#content { width: 100%; margin: 0; float: none; }`
```
Comment (gokulkrishh) [inline CSS/README.md:112]: Nice desc 👍

File: CSS/README.md [hunk 2]
```diff
@@ -102,55 +102,115 @@
...
+
+- [x] Explain how a browser determines what elements match a CSS selector.
+
+  > - CSS Selectors are matched by browser engines from right to left consider `#menu ul li a { color: #00f; }` The browser first checks for `a`, then `li`, then `ul`, and then `#menu`.
+  > - Check matching elements for the key(right-most) selector
+  > - Check if the elements are matching parents for the next selectors
+
+- [x] How would you implement a web design comp that uses non-standard fonts?
```
Comment (gokulkrishh) [inline CSS/README.md]: `comp` ?

File: CSS/README.md [hunk 3]
```diff
@@ -102,55 +102,115 @@
...
+  
+  > - Google web font is a best alternative to non-standard fonts
+  > - If it's a non-standard and custom font then we can use `@font-face` to render it.
+
+- [x] What does `* { box-sizing: border-box; }` do? What are its advantages?
+
+  > - Apply a natural box layout model to all elements, but allowing components to change.
```
Comment (gokulkrishh) [inline CSS/README.md]: May a simpler one ? components are little confusing here.

File: CSS/README.md [hunk 4]
```diff
@@ -102,55 +102,115 @@
...
+
+- [x] How would you implement a web design comp that uses non-standard fonts?
+  
+  > - Google web font is a best alternative to non-standard fonts
+  > - If it's a non-standard and custom font then we can use `@font-face` to render it.
+
+- [x] What does `* { box-sizing: border-box; }` do? What are its advantages?
+
+  > - Apply a natural box layout model to all elements, but allowing components to change. 
+  > - If you set an element width to 300px and if you apply 10px padding then the rendered element on browser would take 300 + leftmargin 20 + rightmargin 20 = total of 340px in width. If you give `box-sizing: border-box` the padding is applied right inside your box model and rendered width would be 300px irrespective of padding / border used.
```
Comment (gokulkrishh) [inline CSS/README.md]: This explain the above. May me mentioned the `content section in the element shrinks` in order to not take more the `300px` width

File: CSS/README.md [hunk 5]
```diff
@@ -102,55 +102,115 @@
...
+  }
+  
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  ```
+
+- [x] How is responsive design different from adaptive design?
+  > - Responsive: There is one basic layout, and it changes responsively to screen changes
+  > - Adaptive: For each possible screen size, there is a distinct layout.
```
Comment (gokulkrishh) [inline CSS/README.md]: 👍

File: CSS/README.md [hunk 6]
```diff
@@ -102,55 +102,115 @@
...
+    background: url(/path/to/my/highreslogo.png) no-repeat;
+    background-size: 200px 400px;
+  }
+}
+```
+
+- [x] The 'C' in CSS stands for Cascading. How is priority determined in assigning styles (a few examples)? How can you use this system to your advantage?
+
+  > CSS priority is determined by [specificity and inheritance](https://www.smashingmagazine.com/2010/04/css-specificity-and-inheritance/).
+  > - Specificity: ID > class, psuedo-class > element, psudo-element
```
Comment (gokulkrishh) [inline CSS/README.md]: Ohh. I didnt know `psuedo-class > element` 💯

**Comment by gokulkrishh** [comment]
Few questions and changes. otherwise 💯 to me

**Comment by logeshpaul** [comment]
@gokulkrishh Addressed PR comments, Going to merge now.

**Comment by gokulkrishh** [comment]
@logeshpaul Sure 👍

**Pull request merged**